ui/src/components/issuers.tsx/IssuerForm.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button, Col, Divider, Form, Input, Row, Select } from "antd";
+import { useState } from "react";
+import IconBack from "src/assets/icons/arrow-narrow-left.svg?react";
+import { AuthBJJCredentialStatus, IssuerFormData, IssuerType } from "src/domain/identifier";
+import { VALUE_REQUIRED } from "src/utils/constants";
+
+export function IssuerForm({
+  onBack,
+  onSubmit,
+}: {
+  onBack: () => void;
+  onSubmit: (formValues: IssuerFormData) => void;
+}) {
+  const [form] = Form.useForm<IssuerFormData>();
+  const [showCredentialStatusField, setShowCredentialStatusField] = useState(false);
+
+  const handleFormChange = (changedValues: Partial<IssuerFormData>) => {
+    if (changedValues.type) {
+      if (changedValues.type === IssuerType.BJJ) {
+        setShowCredentialStatusField(true);
+      } else {
+        setShowCredentialStatusField(false);
+      }
+    }
+  };
+
+  return (
+    <Form form={form} layout="vertical" onFinish={onSubmit} onValuesChange={handleFormChange}>
+      <Form.Item label="Method" name="method" rules={[{ message: VALUE_REQUIRED, required: true }]}>
+        <Input placeholder="Method" />
```

Comment:
Having in the placeholder the same as we have in the label is useless.

Reply:
I will request changes from the designer